### PR TITLE
#9-Update head.html to include Esri global nav

### LINKED ISSUE #9: Integrate live site header + footer
- footer code: https://github.com/aemsites/esri/blob/main/blocks/footer/footer.js
- header code: https://github.com/aemsites/esri/blob/main/blocks/header/header.js

The header and footer code are both the default that comes with the boilerplate the code can be completely rewritten.

### PR DESCRIPTION
**Pull request by tim-white-esri**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #9 

Test URLs:
- Original: https://www.esri.com/en-us/about/about-esri/overview
- Before: https://main--esri--aemsites.hlx.live/en-us/about/about-esri/overview
- After: https://global-nav--esri--aemsites.hlx.live/en-us/about/about-esri/overview

Note: on first load you may need to click a link to generate a CORS acceptence.

### CODE REVIEW

File: head.html [hunk 1]
```diff
@@ -1,6 +1,11 @@
 <meta name="viewport" content="width=device-width, initial-scale=1"/>
 <script src="/scripts/aem.js" type="module"></script>
 <script src="/scripts/scripts.js" type="module"></script>
+<script type="text/javascript">
+    window.gnav_jsonPath = "https://cors-anywhere.herokuapp.com/https://www.esri.com/content/experience-fragments/esri-sites/en-us/site-settings/global-navigation-config/2022-nav-config.25.json";
+</script>
+<script src="https://webapps-cdn-dev.esri.com/CDN/components/global-nav/js/gn.js"></script>
```
Comment (alexcarol) [inline head.html:7]: This doesn't load for me, can you allow it for everyone? If that's an issue, let's discuss alternatives later today.

**Comment by tim-white-esri** [comment]
As a temporary solution, need to set a CORs exception. At least once per day, go to 
https://cors-anywhere.herokuapp.com/corsdemo

Click the button, then site nav should work. 
temp-cors-access

File: blocks/header/header.js [hunk 2]
```diff
@@ -91,6 +91,15 @@ function toggleMenu(nav, navSections, forceExpanded = null) {
...
 export default async function decorate(block) {
+  // Global nav
+  const json = document.createElement('script');
+  json.appendChild(document.createTextNode('window.gnav_jsonPath = 20220-nav-config.25.json'));
+  block.appendChild(json);
+
+  const script = document.createElement('script');
+  script.src = 'https://webapps-cdn.esri.com/CDN/components/global-nav/js/gn.js';
+  block.appendChild(script);
+
```
Comment (alexcarol) [inline blocks/header/header.js:102]: Can you remove the rest of the header code? That's just boilerplate that's only needed if we're implementing the nav in-project.

**Comment by tim-white-esri** [comment]
@alexcarol used the `loadScript` and `loadCSS` to include global nav. I have one const still for the JSON file. Checking to see if we actually need that long term.

File: blocks/header/header.js [hunk 3]
```diff
@@ -1,149 +1,15 @@
...
-  navWrapper.append(nav);
-  block.append(navWrapper);
+  // Global nav
+  const json = document.createElement('script');
+  json.appendChild(document.createTextNode('window.gnav_jsonPath = 20220-nav-config.25.json'));
+  block.appendChild(json);
 }
+
+loadScript('https://webapps-cdn.esri.com/CDN/components/global-nav/js/gn.js');
+loadCSS('https://webapps-cdn.esri.com/CDN/components/global-nav/css/gn.css');
```
Comment (alexcarol) [inline blocks/header/header.js:15]: is there a reason why these two are not inside the "decorate" function?

File: blocks/header/header.js [hunk 4]
```diff
@@ -1,149 +1,15 @@
...
-  isDesktop.addEventListener('change', () => toggleMenu(nav, navSections, isDesktop.matches));
-
-  const navWrapper = document.createElement('div');
-  navWrapper.className = 'nav-wrapper';
-  navWrapper.append(nav);
-  block.append(navWrapper);
+  // Global nav
+  const json = document.createElement('script');
+  json.appendChild(document.createTextNode('window.gnav_jsonPath = 20220-nav-config.25.json'));
+  block.appendChild(json);
```
Comment (alexcarol) [inline blocks/header/header.js:11]: Did you try this instead?
```suggestion
window.gnav_jsonPath = '20220-nav-config.25.json';
```

**Review by alexcarol — APPROVED** [review]
I think it's good to go. There's this warning about unused js, is there a chance it's truly unused, or is just not needed for the initial load?
image

**Pull request merged**